src/configuration.cc

```diff
@@ -39,6 +39,8 @@ constexpr const char kMetadataIngestionDefaultEndpointFormat[] =
 constexpr const int kMetadataIngestionDefaultRequestSizeLimitBytes =
     8*1024*1024;
 constexpr const char kMetadataIngestionDefaultRawContentVersion[] = "0.1";
+constexpr const int kInstanceUpdaterDefaultIntervalSeconds = 60*60;
```

Comment:
An interval of 1 hour? Because GCE instance don't get updated often?

Reply:
Correct. Actually, the bits we care about (the resource id) never change. I thought of actually making it 24 hours, or even larger, because we really only need the initial load on startup. Finally settled on 1 hour, for no good reason.

If we choose to actually send mutable instance metadata, including tags, etc, we would poll this more often (e.g., once a minute), just like any other resource.

Reply:
Makes sense. Thanks for the clarification.